.gitignore

```diff
@@ -23,3 +23,5 @@
 .#*
 /.cache
 /bin/spackc
+*.in.log
```

Comment:
that's probably un-related.

Reply:
This commit is in my other PR (#6551), and this is just rebased on top of it.  Does everyone else not see a bunch of untracked log files introduced in the repo recently?

```console
$ git status
On branch develop
Your branch is up-to-date with 'origin/develop'.
Untracked files:
  (use "git add <file>..." to include in what will be committed)

	spack-cc-binutils-4an3yhe.in.log
	spack-cc-binutils-4an3yhe.out.log
        ...
```

Reply:
i don't see anything like that. I think Spack should not create anything like that.